app/components/balance-donut/component.js

```diff
@@ -43,13 +32,12 @@ class BalanceDonutComponent extends Component {
 
   radius = 100 / (2 * PI);
 
-  @computed("count", "index")
   get style() {
```

Comment:
This is a point where we could consider using [the cached decorator](https://api.emberjs.com/ember/release/functions/@glimmer%2Ftracking/cached), since we do a lot of computation in this getter.